showthread.php

```diff
@@ -561,7 +561,7 @@
 			// Mark the option the user voted for.
 			if(!empty($votedfor[$number]))
 			{
-				$optionbg = "trow2";
+				$optionbg = "poll_votedfor";
```

Comment:
Can we change this to:

```suggestion
				$optionbg = "trow2 poll_votedfor";
```

The upgrade system doesn't manage CSS stylesheet upgrades, so this would just use an unspecified CSS class until users manually updated their themes. If we just add an extra class, it should still allow the same level of flexibility.